### ISSUE TITLE
Error after creating an organization

### ISSUE THREAD

**Issue by hampelm** [detroitledger/gnl_profile#27]
I created an organization manually, which worked, but also displayed this error: 

```
Warning: array_flip() [function.array-flip]: Can only flip STRING and INTEGER values! in DrupalDefaultEntityController->load() (line 178 of /Applications/MAMP/htdocs/grantr/includes/entity.inc).
Warning: array_flip() [function.array-flip]: Can only flip STRING and INTEGER values! in DrupalDefaultEntityController->cacheGet() (line 355 of /Applications/MAMP/htdocs/grantr/includes/entity.inc).
```

**Comment by hampelm** [comment]
Seems to clear up after adding some content so maybe it's a WONTFIX

**Comment by bnchdrff** [comment]
that warning comes from either (or both) of the org_grants views (one lists funded, the other lists received) finding an empty set. i will fix it!

**Comment by bnchdrff** [comment]
also, while i personally care about this specific warning, they won't appear in a non-dev setting

**Comment by bnchdrff** [comment]
fixed in c871791
